[2025-02-08T19:02:43.891+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T19:02:43.912+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-02T00:00:00+00:00 [queued]>
[2025-02-08T19:02:43.922+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-02T00:00:00+00:00 [queued]>
[2025-02-08T19:02:43.923+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T19:02:43.964+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2024-12-02 00:00:00+00:00
[2025-02-08T19:02:43.978+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1636) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T19:02:43.979+0000] {standard_task_runner.py:72} INFO - Started process 1668 to run task
[2025-02-08T19:02:43.984+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2024-12-02T00:00:00+00:00', '--job-id', '269', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_gen.py', '--cfg-path', '/tmp/tmptttgi4gl']
[2025-02-08T19:02:43.989+0000] {standard_task_runner.py:105} INFO - Job 269: Subtask generate_customer_dim_data
[2025-02-08T19:02:44.083+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-02T00:00:00+00:00 [running]> on host 2c1cd59afcc0
[2025-02-08T19:02:44.211+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-02T00:00:00+00:00'
[2025-02-08T19:02:44.213+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T19:02:44.235+0000] {logging_mixin.py:190} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-08T19:02:44.236+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T19:02:44.246+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T19:02:44.247+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, run_id=scheduled__2024-12-02T00:00:00+00:00, execution_date=20241202T000000, start_date=20250208T190243, end_date=20250208T190244
[2025-02-08T19:02:44.278+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T19:02:44.296+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T19:02:44.298+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:40:24.299+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:40:24.332+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-02T00:00:00+00:00 [queued]>
[2025-02-08T20:40:24.367+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-02T00:00:00+00:00 [queued]>
[2025-02-08T20:40:24.407+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:40:24.450+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2024-12-02 00:00:00+00:00
[2025-02-08T20:40:24.500+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2024-12-02T00:00:00+00:00', '--job-id', '171', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_gen.py', '--cfg-path', '/tmp/tmptu4p4nxn']
[2025-02-08T20:40:24.508+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1283) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:40:24.516+0000] {standard_task_runner.py:105} INFO - Job 171: Subtask generate_customer_dim_data
[2025-02-08T20:40:24.517+0000] {standard_task_runner.py:72} INFO - Started process 1300 to run task
[2025-02-08T20:40:24.568+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-02T00:00:00+00:00 [running]> on host 7e959c1a2a8b
[2025-02-08T20:40:24.645+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-02T00:00:00+00:00'
[2025-02-08T20:40:24.647+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:40:24.663+0000] {logging_mixin.py:190} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-08T20:40:24.663+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:40:24.670+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:40:24.670+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, run_id=scheduled__2024-12-02T00:00:00+00:00, execution_date=20241202T000000, start_date=20250208T204024, end_date=20250208T204024
[2025-02-08T20:40:24.693+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:40:24.705+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:40:24.708+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:43:12.912+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:43:12.924+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-02T00:00:00+00:00 [queued]>
[2025-02-08T20:43:12.931+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-02T00:00:00+00:00 [queued]>
[2025-02-08T20:43:12.932+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:43:12.942+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2024-12-02 00:00:00+00:00
[2025-02-08T20:43:12.950+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=619) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:43:12.951+0000] {standard_task_runner.py:72} INFO - Started process 629 to run task
[2025-02-08T20:43:12.950+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2024-12-02T00:00:00+00:00', '--job-id', '461', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_gen.py', '--cfg-path', '/tmp/tmp_fqc5rp0']
[2025-02-08T20:43:12.952+0000] {standard_task_runner.py:105} INFO - Job 461: Subtask generate_customer_dim_data
[2025-02-08T20:43:12.989+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-02T00:00:00+00:00 [running]> on host 86932746817d
[2025-02-08T20:43:13.056+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-02T00:00:00+00:00'
[2025-02-08T20:43:13.057+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:43:13.072+0000] {logging_mixin.py:190} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-08T20:43:13.073+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:43:13.080+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:43:13.080+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, run_id=scheduled__2024-12-02T00:00:00+00:00, execution_date=20241202T000000, start_date=20250208T204312, end_date=20250208T204313
[2025-02-08T20:43:13.125+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:43:13.138+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:43:13.140+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
